### PR TITLE
Add a new GHE district test with new network template models

### PR DESCRIPTION
**Pull request by JingWang-CUB**
#### Any background context you want to provide?
URBANopt GHP 

#### What does this PR accomplish?
Add a new 5G district test which tests a full-scale DES with a single GHE and multiple timeseries loads.

#### How should this be manually tested?
poetry run pytest -m 'not compilation and not simulation' tests/model_connectors/test_district_single_ghe.py

#### What are the relevant tickets?
#547

### CODE REVIEW

**Comment by JingWang-CUB** [comment]
@vtnate , I found a potential bug in the time series load template, where the filNam parameter is having backslashes in the path, causing Modelica to not be able to read the mos files. See screenshot below. The same error can also be found in test_time_series_5g (I guess all time series load related tests have it). Could you take a look at this when you have time? Thanks!
 
image

My PR should be ready for your review once this is fixed.

**Comment by JingWang-CUB** [comment]
Hi @vtnate , looks like my test didn't pass the pytest. Could you let me know how to fix that? The local tests have all passed.

**Comment by JingWang-CUB** [comment]
@vtnate , the pytests have passed but the CI is failing installing some Windows dependencies. Could you please fix those? Thanks.

File: geojson_modelica_translator/model_connectors/couplings/5G_templates/NetworkDistributionPump_NetworkDistributionPump/ComponentDefinitions.mopt
Comment (vtnate) [inline geojson_modelica_translator/model_connectors/couplings/5G_templates/NetworkDistributionPump_NetworkDistributionPump/ComponentDefinitions.mopt:1]: Very well done with the couplings! I love it!

**Review by vtnate — APPROVED** [review]
That modelica-path was painful, thank you for working through it! LGTM.

**Comment by JingWang-CUB** [comment]
@mitchute , this PR is ready for your review, thanks!

**Review by mitchute — APPROVED** [review]

**Pull request merged**